workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/ComposeMultiRendering.kt

```diff
@@ -0,0 +1,22 @@
+package com.squareup.workflow1.ui.compose
+
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.visual.MultiRendering
+import com.squareup.workflow1.visual.VisualEnvironment
+import com.squareup.workflow1.visual.VisualHolder
+
+@WorkflowUiExperimentalApi
+public class ComposeMultiRendering : MultiRendering<Unit, ComposableLambda>() {
```

Comment:
is `MultiRendering` the same as this "Aggregated visual factory" concept in [helios' writeup](https://github.com/squareup/helios-workflow-proposal)?

Reply:
It's more that `MultiRendering` subclasses are the things that use the aggregated factory. They're the shared code for the job that `WorkflowViewStub` and `WorkflowRendering()` both perform:

* holding on to a `var rendering`
* creating the visual for it (via the aggregated thing)
* updating the rendering and
  * updating the existing visual if it's compatible
  * or creating a new visual

The aggregated visual factory is the thing that includes all the leaf compose factories, all the leaf view factories, and the glue code that can convert one type to the other type.